### DISCUSSION TITLE
Experiment identification tag

### DISCUSSION THREAD

**Discussion by LucasGCespedes** [NicWir/QurvE#17] [Ideas]
Hi all,

It would be very useful for me if there is somehow to add a tag to id different experiments in same .csv file custom input. It would be nice to compare same conditions (strains, medium, treatments) in different experiment batches. 

Best,

**Comment by NicWir** [comment]
A low-key solution could be to simply add an experiment-identifying string
to your sample descriptions (called "condition" inside the app).

Example:

"StrainA_carbonsourceA_exp1", StrainA_carbonsourceB_exp1",
StrainA_carbonsourceB_exp2"


Then, you could filter for the respective experiment by using the Search
fields for "Summary" tables or the group plots.

If that doesn't suffice, I'd appreciate it if you could describe a little
what your desired outcome would be. What feature would you like to have
added specifically that could not be solved by the above-mentioned solution?

Thanks a lot for using QurvE and for reaching out! I want this tool to be
as much of a help to the community as possible :-)

Best regards,

Nicolas
